### PR TITLE
Set chunk size in MB (500 by default)

### PR DESCRIPTION
**Pull request by alejoe91**

### CODE REVIEW

**Comment by alejoe91** [comment]
@samuelgarcia something like this shoulw work :)

**Comment by alejoe91** [comment]
can we merge this?

File: spikeextractors/extraction_tools.py [hunk 1]
```diff
@@ -267,25 +267,54 @@ def write_to_binary_dat_format(recording, save_path, time_axis=0, dtype=None, ch
...
     '''
     save_path = Path(save_path)
     if save_path.suffix == '':
         # when suffix is already raw/bin/dat do not change it.
         save_path = save_path.parent / (save_path.name + '.dat')
 
-    if chunk_size is None:
+    if chunk_size is not None or chunk_mb is not None:
+        if time_axis == 1:
+            print("Chunking disabled due to 'time_axis' == 1")
```
Comment (samuelgarcia) [inline spikeextractors/extraction_tools.py]: Good idea.

**Comment by samuelgarcia** [comment]
This is cool.
The way is written is a bit complicated.
the cascade of if else could only serve to get the chunk_size (int or None) and then there would be only one exolicit loop.
But it is ok.

**Comment by alejoe91** [comment]
I agree :) I'll simplify it

**Comment by samuelgarcia** [comment]
:)

**Pull request merged**